ipydb/magic.py

```diff
@@ -185,24 +193,27 @@ def sql(self, args='', cell=None):
             params = self.shell.user_ns.get(args.params, {})
         if args.multiparams:
             multiparams = self.shell.user_ns.get(args.multiparams, [])
-        result = self.ipydb.execute(sql, params=params,
+        cursor = self.ipydb.execute(sql, params=params,
                                     multiparams=multiparams)
+        
+        if not cursor:
+            return None
+        if not cursor.returns_rows:
+            s = 's' if cursor.rowcount != 1 else ''
+            print("%i row%s affected" % (cursor.rowcount, s))
+
+        if args.pandas:
+            return self.ipydb.build_dataframe(cursor)
         if args.ret:
-            return result
-        if result and result.returns_rows:
```

Comment:
I think we still need these checks around returns_rows. These come into play when you run an update/delete/insert statement. I have added a test to master for running an insert statement. This will break in your pull request. Further, the '%i rows affected.' is useful output when you are running update/insert/delete statements. Could you put those back in?